src/collector/layer-zero/layer-zero.worker.ts

```diff
@@ -0,0 +1,617 @@
+        log: Log,
+        parsedLog: LogDescription,
+    ): Promise<void> {
+        debugger;
+
+        try {
+            const {
+                encodedPayload,
+                options,
+                sendLibrary
+            } = parsedLog.args as unknown as PacketSentEvent.OutputObject;
+            const packet = this.decodePacket(encodedPayload);
+            const srcEidMapped = this.layerZeroChainIdMap[packet.srcEid];
+            const dstEidMapped = this.layerZeroChainIdMap[packet.dstEid];
+
+            this.logger.debug(
+                { transactionHash: log.transactionHash, packet, options, sendLibrary },
+                'PacketSent event found: log details.',
+            );
+
+            if (srcEidMapped === undefined || dstEidMapped === undefined) {
+                this.logger.debug(
+                    {
+                        transactionHash: log.transactionHash,
+                        packet,
+                        options,
+                        sendLibrary,
+                    },
+                    'Skipping packet: unsupported srcEid/dstEid.',
+                );
+                return;
+            }
+
+            const decodedMessage = ParsePayload(packet.message);
+            if (decodedMessage === undefined) {
+                throw new Error('Failed to decode message payload.');
+            }
+            if (
+                paddedTo0xAddress(packet.sender).toLowerCase() ===
+                this.incentivesAddresses[srcEidMapped]
+            ) {
+                this.logger.info(
+                    { sender: packet.sender, message: packet.message },
+                    'Processing packet from specific sender: sender and message details.',
+                );
+
+                try {
+                    const transactionBlockNumber =
+                        await this.resolver.getTransactionBlockNumber(log.blockNumber);
+                    await this.store.setAmb(
+                        {
+                            messageIdentifier: decodedMessage.messageIdentifier,
+                            amb: 'layer-zero',
+                            sourceChain: srcEidMapped.toString(),
+                            destinationChain: dstEidMapped.toString(),
+                            sourceEscrow: packet.sender,
+                            payload: decodedMessage.message,
+                            recoveryContext: '0x',
+                            blockNumber: log.blockNumber,
+                            transactionBlockNumber,
+                            blockHash: log.blockHash,
+                            transactionHash: log.transactionHash,
+                        },
+                        log.transactionHash,
+                    );
+
+                    this.logger.info(
+                        { transactionHash: log.transactionHash },
+                        'Primary AMB message created using setAmb: transactionHash details.',
+                    );
```

Comment:
Make use of the following log statement to be consistent with the Wormhole collector implementation. Also, include the `payloadHash` on the log, and move the full log statment after the `await this.store.setPayload(...)` call. This way we only emit 1 log per collected message.
```suggestion
                    this.logger.info(
                        {
                            messageIdentifier: decodedMessage.messageIdentifier,
                            transactionHash: log.transactionHash,
                            payloadHash
                        },
                        'Collected message.',
                    );
```